app/tests/unit/specs/components/account/__snapshots__/EmailAuthentication.spec.js.snap

```diff
@@ -1,22 +1,37 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`EmailAuthentication should show the EmailAuthentication page 1`] = `
-<tmpage-stub
-  data-title="email-authentication"
-  emptysubtitle=""
-  emptytitle="No data"
-  loaderpath=""
+<sessionframe-stub
+  hideback="true"
+  icon=""
 >
+  <h2
+    class="session-title"
+  >
+    
+    Magic link success!
+  
+  </h2>
+   
   <div
-    class="card"
+    class="session-main"
   >
-    <h1
-      class="authentication-title"
+    <p
+      class="session-subtitle"
     >
       
-      You are now signed in! 🚀
+      You're now signed in to Lunie with your email address. Head over to the
+      notifications page to see some of your recent events.
     
-    </h1>
+    </p>
+     
+    <tmbtn-stub
+      centered="true"
```

Comment:
why do we need to center here and in other places not?

Reply:
mostly about visual symmetry. when a modal has a form or some inputs the button can go on the right to imply progress and also to balance the weight of the elements visually. 

for a simple message like this one, with one call to action and centered title and text - i think the centered button is the strongest layout.